internal/logger/logger.go

```diff
@@ -229,6 +229,13 @@ func getSource(level int) string {
 	return ""
 }
 
+// HashString - return the highway hash of the passed string
```

Comment:
This was not cherry-picked properly, please refer to the change here https://github.com/minio/minio/pull/16853/files